android/assets/i18n/messages.properties

```diff
@@ -38,35 +38,35 @@ game-select.high-score=High Score
 
 unimplemented-game.title=More Games Coming Soon!
 unimplemented-game.description=Know of a good game to work on next?\nVote for it on GitHub.
-unimplemented-game.next-game-dev=When we get time to add another game,\nthose with many thumbs up will most likely come next.
+unimplemented-game.next-game-dev=When we get time to add another game,\nthose with many upvotes will most likely come next.
```

Comment:
Strictly speaking, there is no such thing as "upvotes" in GitHub. You can react with many different emoji's, but it is procedurally easier for me just to look at thumbs up (because an individual user can "vote" with many different reactions).